libpod/image.go

```diff
@@ -56,11 +72,20 @@ type ImageFilter func(*storage.Image) bool
 // Signature validation will be performed if the Runtime has been appropriately
 // configured
 func (r *Runtime) PullImage(imgName string, allTags bool, reportWriter io.Writer) error {
```

Comment:
Do we want to allow overriding of the default SignaturePolicy with pull? Right now, we use the default in the Runtime. Are there cases where we need to override it?

Reply:
pull doesn't have a flag to override it, so I think we just use the default.

Reply:
Lets add it to the interface and then just pass the default.